website/src/components/Submission/FileUpload/SequenceEntryUploadComponent.tsx

```diff
@@ -0,0 +1,233 @@
+                    )}
+                    The documentation pages contain more details on the required{' '}
+                    <a href={metadataFormatDocsUrl} className='text-primary-700 opacity-90'>
+                        metadata format
+                    </a>{' '}
+                    including a list of all supported metadata. You can download a{' '}
+                    <a href={routes.metadataTemplate(organism, action, 'tsv')} className='text-primary-700 opacity-90'>
+                        TSV
+                    </a>
+                    {' or '}
+                    <a href={routes.metadataTemplate(organism, action, 'xlsx')} className='text-primary-700 opacity-90'>
+                        XLSX
+                    </a>{' '}
+                    template with column headings for the metadata file.
+                </p>
+
+                {isMultiSegmented && (
+                    <p className='text-gray-400 text-xs mt-3'>
+                        {organism.toUpperCase()} has a multi-segmented genome. Please submit one metadata entry with a
+                        unique <i>submissionId</i> for the full multi-segmented sample, e.g. <b>sample1</b>. Sequence
+                        data should be a FASTA file with each header indicating the <i>submissionId</i> and the segment,
+                        i.e.{' '}
+                        {referenceGenomeSequenceNames.nucleotideSequences.map((name, index) => (
+                            <span key={index} className='font-bold'>
+                                sample1_{name}
+                                {index !== referenceGenomeSequenceNames.nucleotideSequences.length - 1 ? ', ' : ''}
+                            </span>
+                        ))}
+                        .
+                    </p>
+                )}
+
+                <p className='text-gray-400 text-xs mt-3'>
+                    Files can optionally be compressed, with the appropriate extension (<i>.zst</i>, <i>.gz</i>,{' '}
+                    <i>.zip</i>, <i>.xz</i>). For more information please refer to the{' '}
+                    <a href={dataUploadDocsUrl} className='text-primary-700 opacity-90'>
+                        help pages
+                    </a>
+                    .
+                </p>
+
+                {(organism.startsWith('not-aligned-organism') || organism.startsWith('dummy-organism')) &&
+                    action === 'submit' && (
+                        <DevExampleData
+                            setExampleEntries={setExampleEntries}
+                            exampleEntries={exampleEntries}
+                            handleLoadExampleData={handleLoadExampleData}
+                            dataIsLoaded={!!metadataFile && (!enableConsensusSequences || !!sequenceFile)}
+                        />
+                    )}
+            </div>
+            <form className='sm:col-span-2'>
+                <div className='flex flex-col lg:flex-row gap-6 justify-end'>
```

Comment:
```suggestion
                <div className='flex flex-col lg:flex-row gap-6'>
```

IMO this should be lined up with the UI below as on main